src/app/index.html

```diff
@@ -1 +1 @@
-<my-home></my-home>
+<my-home role="main"></my-home>
```

Comment:
This will also encompass the `app-nav` component, right? My gut is that we should either include a `<main>` tag or a `<div role="main">`  in the **home.component.html** file...

```
<app-nav>
</app-nav>
<main>
  // ... 
</main>
```

https://github.com/blackbaud/skyux-template/blob/master/src/app/home.component.html#L2

Reply:
Good catch.  Let me know if the current approach satisfies your concerns.